docs/fastdds/dds_layer/core/policy/standardQosPolicies.rst

```diff
@@ -211,8 +211,8 @@ There are four possible values (see |DurabilityQosPolicyKind-api|):
 * |TRANSIENT_LOCAL_DURABILITY_QOS-api|: When a new DataReader joins, its History is filled with past samples.
 * |TRANSIENT_DURABILITY_QOS-api|: When a new DataReader joins, its History is filled with past samples, which are stored
   on persistent storage (see :ref:`persistence_service`).
-* |PERSISTENT_DURABILITY_QOS-api|: (`Not Implemented`): All the samples are stored on a permanent storage, so that they
-  can outlive a system session.
+* |PERSISTENT_DURABILITY_QOS-api|: (`Not Implemented`, currently behaves as |TRANSIENT_DURABILITY_QOS-api|):
+  All the samples are stored on a permanent storage, so that they can outlive a system session.
```

Comment:
Assuming my suggestion [here](https://github.com/eProsima/Fast-DDS/pull/5224#pullrequestreview-2317683403) is accepted, we can either join TRANSIENT and PERSISTENT in a single bullet or make them have the same text.